components/index/VolunteerHero.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div class="c-hero">
+    <h1 class="c-hero__header1">{{ $t('Volunteer at a Dojo near you') }}</h1>
+    <div class="c-hero__images">
+      <div class="c-hero__image">
+        <img src="https://picsum.photos/480/320?image=1" />
+      </div>
+      <div class="c-hero__image">
+        <img src="https://picsum.photos/480/320?image=2" />
+      </div>
+      <div class="c-hero__image">
+        <img src="https://picsum.photos/480/320?image=3" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+  @import '~/assets/_variables.scss';
+
+  .c-hero {
+    padding: 0 0 80px 0;
+    background-image:
+      radial-gradient(transparent 50%, transparent 100%),
+      linear-gradient(to bottom, $cd-purple 0%, $cd-purple-gradient 100%);
+    background-size: 200% 300px, 100%;
```

Comment:
Since you don't have multiple backgrounds anymore, there should be no need for `background-size`, `background-position` or `background-repeat` since it will default to 100% size with top right positioning.